kaggle_environments/envs/werewolf/werewolf.json

```diff
@@ -0,0 +1,286 @@
+{
+    "name": "werewolf",
+    "version": "1.0.0",
+    "title": "Werewolf",
+    "description": "A social deduction game.",
+    "agents": [7, 8, 9, 10, 11, 12, 13, 14, 15],
+    "configuration": {
+        "episodeSteps": { "type": "integer", "default": 1000 },
+        "actTimeout": { "type": "number", "default": 3 },
+        "runTimeout": { "type": "number", "default": 1800 },
+        "seed": { "type": "integer", "description": "Seed to use for episodes" },
+        "night_elimination_reveal_level": {
+            "type": "string",
+            "enum": ["no_reveal", "team", "role"],
+            "default": "role",
+            "description": "During night elimination, the moderator should reveal this level of information about the eliminated player."
+        },
+        "day_exile_reveal_level": {
+          "type": "string",
+            "enum": ["no_reveal", "team", "role"],
+            "default": "role",
+            "description": "During day exile, the moderator should reveal this level of information about the exiled player."
+        },
+        "agents": {
```

Comment:
@hannw This part is a little up in the air as we're discussing, but fine to merge for now